api/definitions.yaml

```diff
@@ -253,8 +253,13 @@ LimitedProfile:
     attributes.
   type: object
   properties:
+    sender_allowed:
+      type: boolean
+      default: true
```

Comment:
what does the default do in our logic?

Reply:
also worth adding a description since this is shared with the PAs

Reply:
removed default and added description